recipe/deploy/shared.php

```diff
@@ -23,7 +23,9 @@
     }
 
     foreach (get('shared_dirs') as $dir) {
+        var_dump($dir);
```

Comment:
Vad dump?

Reply:
this was temporary code to check my code. Now, it was removed.